### PR TITLE
Ana/add polkadot e2e test script

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Mini PR. Also thinking maybe it is better to add the `npx env-cmd` in front of the nightwatch command

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f4e8c12a4814e08ee29532f--lunieio.netlify.app

**Comment by jbibla** [comment]
this isnt a mini PR - it's a normal sized PR - all the others are big PRs

**Comment by Bitcoinera** [comment]
E2E are failing a lot. Can they be broken again?

File: app/package.json [hunk 1]
```diff
@@ -20,8 +20,9 @@
     "test": "npm run test:unit && npm run test:e2e",
     "test:coverage": "live-server ./coverage/lcov-report",
     "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
-    "test:e2e:browserstack": "nightwatch -c tests/e2e/browserstack.conf.js -e windows",
+    "test:e2e:browserstack": "npx env-cmd nightwatch -c tests/e2e/browserstack.conf.js -e windows --fe='https://app.lunie.io' --api=https://staging-api.lunie.io",
```
Comment (faboweb) [inline app/package.json:23]: this would test the production version not the current version

Comment (Bitcoinera) [reply app/package.json:23]: how to make it dynamic? Is there a way?

Comment (faboweb) [reply app/package.json:23]: instead of doing this script, just document in the README if it is not already

Comment (Bitcoinera) [reply app/package.json:23]: OK, I will add it

**Review by faboweb — APPROVED** [review]

**Review by faboweb — APPROVED** [review]

**Pull request merged**